packages/@o3r/core/schematics/ng-add/schema.json

```diff
@@ -15,79 +15,81 @@
+      "x-prompt": "Activate prefetch builder to generate prefetcher JavaScript file (see https://github.com/AmadeusITGroup/ngx-prefetch)?"
     },
     "enablePlaywright": {
       "type": "boolean",
       "default": true,
       "description": "Install playwright",
-      "x-prompt": "Activate playwright as E2E test system ?"
+      "x-prompt": "Activate playwright as E2E test system?"
     },
     "enableCustomization": {
       "type": "boolean",
       "default": true,
       "description": "Install Otter customization",
-      "x-prompt": "Add Otter customization mechanism to override presenter and injectable for external application"
+      "x-prompt": "Add Otter customization mechanism to allow presenter and injectable override by external custom application"
     },
     "enableAnalytics": {
       "type": "boolean",
       "default": true,
       "description": "Install Otter analytics",
-      "x-prompt": "Activate Otter analytics ?"
+      "x-prompt": "Activate Otter Analytics to gather and report analytics to specific providers (such as Google Analytics, Adobe Target, ...)?"
     },
     "enableCms": {
       "type": "boolean",
       "default": true,
       "description": "Activate full CMS compatibility",
-      "x-prompt": "Add otter packages ('@o3r/{localization,styling,components,configuration}') to make the aplication CMS compatible ?"
+      "x-prompt": "Add all mandatory Otter packages (@o3r/localization, @o3r/styling, @o3r/components and @o3r/configuration) to make the application CMS compatible?"
     },
     "enableStyling": {
       "type": "boolean",
       "default": true,
       "description": "Install Otter theming and styling extractor",
-      "x-prompt": "Add Otter styling setup?"
+      "x-prompt": "Add Otter theming and styling mechanism (included in CMS package)?"
     },
     "enableConfiguration": {
       "type": "boolean",
       "default": true,
       "description": "Install otter configuration package and config extractor setup",
-      "x-prompt": "Add Otter configuration setup ?"
+      "x-prompt": "Add Otter configuration allowing application and component runtime configuration (included in CMS package)?"
     },
     "enableRulesEngine": {
       "type": "boolean",
       "default": false,
       "description": "Install otter rules-engine package",
-      "x-prompt": "Add Otter rules-engine setup ?"
+      "x-prompt": "Add Otter rules-engine to execute runtime custom rules defined in external json file (see https://github.com/AmadeusITGroup/otter/tree/main/docs/rules-engine)?"
     },
     "enableLocalization": {
       "type": "boolean",
       "default": true,
       "description": "Localization generation setup and localization extractor",
-      "x-prompt": "Add Otter localization setup ?"
+      "x-prompt": "Add Otter localization allowing externally configured runtime multi languages (included in CMS package)?"
```

Comment:
Honestly, I find this one confusing for someone that is external to the project.